### ISSUE TITLE
Missing Transcript check won't find non-sibling transcripts

### ISSUE THREAD

**Issue by pattonwebz** [equalizedigital/accessibility-checker#1254]
If the transcript is not a sibling of the video, or not a sibling of the parent, and the wrapping container is not a select type of element then the transcript won't be found.

But transcripts that are near should be found, even when not direct siblings.
